### PR TITLE
[SENT 30/10/17] Digest

### PR DESCRIPTION
**Pull request by nvgrw**
Pls review thx

### CODE REVIEW

File: markdown/2017-10-30.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+Founders Factory team during their own robotic restaurant project. Teams will
+be given two days to come up with and build out solutions before presenting
+them to a star studded panel of judges. The hackathon will provide great
+opportunities to explore, network and learn, whilst also gaining exposure and
+recognition from your peers, the Founders Factory team and other experts.
+ 
+Let your friends know you're coming on the Facebook event
+[here](https://www.facebook.com/events/1410061225709920/)! To find out more
+and to register, either as an individual or team, please click the button
+below.Please note, if applying as a team, we recommended that teams are no
```
Comment (plietar) [inline markdown/2017-10-30.md]: Space missing between below and Please

File: markdown/2017-10-30.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+on member polls, the top fields of interest were: Security/Cryptography,
+Artificial Intelligence and Programming Languages.
+
+First and foremost, our own DoCSoc reading group in Programming Languages
+will be held in Huxley 145 on Tuesday, October 31 at 1-2pm. Please arrive 5
+minutes early if possible due to the large number of speakers. Cutting edge
+research papers published at leading Programming Languages conferences, such
+as OOPSLA, will be presented by their authors. You can find details about the
+speakers and the papers at <https://docsoc.co.uk/papers>. *This event is
+aimed at 3rd and 4th year students, however everyone interested in
```
Comment (plietar) [inline markdown/2017-10-30.md]: Maybe include MScs as well?

File: markdown/2017-10-30.md [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+programming languages is welcome to attend.*
+
+As for Security/Cryptography and Artificial Intelligence, we have partnered
+with Ben Livshits and Marc Deisenroth to advertise their own reading groups.
+You can find the links on the DoCSoc website. *These events are aimed at
```
Comment (plietar) [inline markdown/2017-10-30.md]: There's nothing on the website.
@egnwd?

File: markdown/2017-10-30.md (hunk 2 above)
Comment (nvgrw) [reply markdown/2017-10-30.md]: @qvks

**Comment by nvgrw** [comment]
I'm ready to send this off.  [Latest draft PDF](https://github.com/icdocsoc/emails/files/1427205/draft2.pdf)

**Pull request merged**